### PR TITLE
tests: fix test cases initialize and cleanup

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: testing/opae-c/test_enum_c.cpp [hunk 1]
```diff
@@ -97,8 +97,9 @@ class enum_c_p : public ::testing::TestWithParam<std::string> {
     }
 
     int value;
-    std::string cmd = "ls -l /sys/class/fpga/intel-fpga-dev* | "
-                      "wc -l";
+    std::string cmd =
+        "(ls -l /sys/class/fpga*/region*/*fme*/dev || "
+        "/sys/class/fpga*/*intel*) |  (wc -l)";
```
Comment (ibarraza) [inline testing/opae-c/test_enum_c.cpp]: Isn't there supposed to be another ls in front of /sys/class/fpga*/\*intel*?

File: testing/xfpga/test_enum_c.cpp [hunk 2]
```diff
@@ -84,8 +84,9 @@ class enum_c_p : public ::testing::TestWithParam<std::string> {
     }
 
     int value;
-    std::string cmd = "ls -l /sys/class/fpga/intel-fpga-dev* | "
-                      "wc -l";
+    std::string cmd =
+        "(ls -l /sys/class/fpga*/region*/*fme*/dev || "
+        "/sys/class/fpga*/*intel*) |  (wc -l)";
```
Comment (ibarraza) [inline testing/xfpga/test_enum_c.cpp]: Isn't there supposed to be another ls in front of /sys/class/fpga*/\*intel*?

File: testing/opae-c/test_enum_c.cpp (hunk 1 above)
Comment (anandaravuri) [reply testing/opae-c/test_enum_c.cpp]: Fixed

File: testing/xfpga/test_enum_c.cpp (hunk 2 above)
Comment (anandaravuri) [reply testing/xfpga/test_enum_c.cpp]: fixed

**Pull request merged**